lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/domain/dva/request/DvaPayload.java

```diff
@@ -26,11 +26,11 @@ public class DvaPayload {
 
     @JsonProperty private UUID correlationId;
     @JsonProperty private UUID requestId;
+    @JsonProperty private String issuerId;
     @JsonProperty private String timestamp;
-    @JsonProperty private String licenceNumber;
-    @JsonProperty private String driverNumber;
+    @JsonProperty private String driverLicenceNumber;
```

Comment:
This needs rebased I dont think we should be seeing these changes on this PR

Reply:
Corrected.